strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/storage/validation/version/PypiCanonicalVersionValidator.java

```diff
@@ -123,4 +130,6 @@ private static String getCanonicalVersionFormat()
                DEVELOPMENTAL_RELEASE_FORMAT + "?" + LOCAL_VERSION_IDENTIFIER_FORMAT + "?)$";
     }
 
+    private BiPredicate<Pattern, String> versionMatches = (pattern, version) -> pattern.matcher(version).matches();
```

Comment:
Please, place this private member next to the others.

Reply:
Done. Thanks.